### PR TITLE
Handle external id for users

### PR DESCRIPTION
**Pull request by tkrille**
This PR implements 2 features:

1. Allow setting of `external_id` for users
2. Add button to copy user id into external id field

Fixes: #75

### CODE REVIEW

File: src/main/resources/templates/user/editUser.html [hunk 1]
```diff
@@ -12,6 +12,14 @@
             th:src="@{/js/bootstrap-datetimepicker.min.js}"></script>
     <script type="text/javascript" src="../../js/multivalue.js" th:src="@{/js/multivalue.js}"></script>
     <script type="text/javascript" src="../../js/extension.js" th:src="@{/js/extension.js}"></script>
+    <script type="text/javascript">
+        $(document).ready(function () {
+            $("#externalId_copyFromId").click(function (e) {
```
Comment (dacrome) [inline src/main/resources/templates/user/editUser.html]: There aren't any HTML ID naming conventions, but I think you shouldn't mix camel case and underscores. I prefer to use hyphens.

Comment (tkrille) [inline src/main/resources/templates/user/editUser.html]: Done. It's camelCase now.

**Comment by tkrille** [comment]
Updated

**Comment by dacrome** [comment]
works like a charm!

**Pull request merged**